FordLabs/peoplemover
# Test PeopleMover Using Docker

## How to test
Starting from no environment, execute the steps in the Readme for "Test out PeopleMover". Confirm that you have a working test environment.
